src/Intl/Idn/bootstrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Intl\Idn as p;
+
+if (!function_exists('idn_to_ascii')) {
+    define('IDNA_DEFAULT', 0);
```

Comment:
what about all the other `IDNA` constants allowing to configure it ?

Reply:
I just added them this morning